packages/shared/components/atoms/MediaDisplay.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="typescript">
+    import { MimeType, ParentMimeType } from '@core/nfts'
+
+    export let Media: HTMLImageElement | HTMLVideoElement
+    export let src: string
+    export let expectedType: MimeType
+    export let classes: string
```

Comment:
```suggestion
    export let classes: string = ''
```

Reply:
Because else we get a warning that classes is required

Reply:
ahh only when your vs code needs resetting but sure